### PR TITLE
Unique

### PR DESCRIPTION
**Pull request by fqiu7**
**Describe your changes**
Add unique function

**Testing performed**
Integration test

### CODE REVIEW

File: expr/func_unique.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	}
+	seenNames := make(map[string]bool)
+	var uniqueSeries []models.Series
+	for _, serie := range series {
+		if _, ok := seenNames[serie.Target]; !ok {
+			seenNames[serie.Target] = true
+			uniqueSeries = append(uniqueSeries, serie)
+		}
+	}
+	fmt.Println("New UNIQUE func called !!!")
```
Comment (shanson7) [inline expr/func_unique.go]: Should be removed

**Review by shanson7 — CHANGES_REQUESTED** [review]
Make sure to update docs/graphite.md

File: expr/func_unique.go (hunk 1 above)
Comment (fqiu7) [reply expr/func_unique.go]: removed

Comment (fqiu7) [reply expr/func_unique.go]: I also updated docs/graphite.md

**Review by shanson7 — APPROVED** [review]
Nice work! Leave this PR open and approved as a reminder for me to merge it in to our next release branch. Submit a new PR against grafana/metrictank

**Comment by fqiu7** [comment]
sure I will leave this open and submit a new PR against grafana/metrictank. thanks.

**Pull request closed without merging**